charts/prometheus-cloudwatch-exporter/values.yaml

```diff
@@ -202,3 +202,8 @@ ingress:
 
 securityContext:
   runAsUser: 65534  # run as nobody user instead of root
+
+# Leverage a PriorityClass to ensure your pods survive resource shortages
+# ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/
+# PriorityClass: system-cluster-critical
```

Comment:
Do we need this line? Should it be PriorityClassName?

Reply:
That line is referring to the `PriorityClass` resource.
Sure, I can remove it if is not clear enoght

Reply:
I was referring to `# PriorityClass: system-cluster-critical`. It's not used in templates.

You are using `priorityClassName` in templates. That's why I asked if it should be renamed in the comment.

Reply:
Done 👍